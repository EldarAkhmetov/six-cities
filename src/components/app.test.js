import React from 'react';
import Enzyme, {shallow} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import App from './app';

Enzyme.configure({adapter: new Adapter()});


it(`App correctly renders after launch`, () => {
  const tree = shallow(<App />);

  expect(tree).toMatchSnapshot();
});
