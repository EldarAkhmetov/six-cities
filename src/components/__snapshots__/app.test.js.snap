// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App correctly renders after launch 1`] = `ShallowWrapper {}`;
